routes/apiRoutes.js

```diff
@@ -0,0 +1,43 @@
+import express from 'express'
+const router = express.Router()
+import BlogModel from '../model/BlogModel.js'
+
+// API route to get all blogs
+router.get('/blog', (req, res, next) => {
+	BlogModel.find().then((blogs) => {
+		res.json(blogs)
+	})
+})
+
+// Route to post a new blog
+router.post('/blog', (req, res, next) => {
+	BlogModel.create({
+		title: req.body.title,
+		body: req.body.body
+	}).then((newBlog) => {

```

Comment:
for now,  this a good draft. later, we will need to a validation & security check.
Clarence & I can give you an update on this